sandbox/anvi-script-reformat-fasta

```diff
@@ -133,6 +146,10 @@ def reformat_FASTA(args):
         else:
             output.store(fasta, split = False)
 
+    if args.export_gap_counts_table:
+        df = pd.DataFrame(gaps_info_list, columns=["header", "num_gaps", "sequence"])
+        df.to_csv(args.export_gap_counts_table + ".tsv", sep='\t', index=False)
+
```

Comment:
Users you want this info probably don't want a reformatted FASTA, they first just want this table so they can decide the `--max-gaps` they want to use. Am I understanding right?

Reply:
Yeah it's just to quickly examine the distribution of gap counts per sequence. I actually left the sequences column in by accident because I was double-checking in R that the gap counts were correct, but left it because I thought it might be useful for the user. Should I take out the sequence column?

Reply:
Well, it is totally up to you. But keep in mind that you don't control how big the user's sequences are. Then again it is an MSA so it can't be **that** crazy big.

Reply:
That's I good point, I removed the column.